app/lib/excel_sanitizer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# String manipulation of data fields that are to be exported
+# Not intended for to protect against any threat vector through Excel
```

Comment:
extra "for" here

Reply:
https://github.com/SaraAlert/SaraAlert/pull/718/commits/5445da655791b866b9f5989c01935c37b1fa20d1